### DISCUSSION TITLE
Report results with new opencl-runtime

### DISCUSSION THREAD

**Discussion by tpkessler** [rocm-arch/rocm-arch#711] [Support] [answered]
As mentioned in #710 the new `rocm-opencl-runtime 5.1.0` does not work on my machine. The binary version installed via `opencl-amd` works. So it's definitely a software and not a hardware problem. Can some one please build the new runtime and report their results in this thread?

I haven't pushed the new code to the AUR yet, so you have to use the PKGBUILD directly from this repo.

**Comment by wsippel** [comment]
Just posted in your bug report, but I tested your PKGBUILD and run into the same issue. The problem is #705 - comgr fails to load as it can't find libLLVM-14git.so. This breaks opencl-runtime and both HIP applications I tested (Blender 3.2 and hashcat). I installed hip-runtime-amd 5.1 with a modified PKGBUILD. 

I guess adding /opt/rocm/llvm/lib to the library search path could lead to conflicts, so maybe compiling comgr with RPATH might be an option?

**Reply by tpkessler** [reply]
I found that reverting recent changes to `rocm-llvm` solves this issue. Thank you for your help. Much appreciated!

**Comment by tpkessler** [comment, marked as answer]
Fixed with [6287da5](https://github.com/rocm-arch/rocm-arch/commit/6287da54667b77d0d32a8354ad58b71292e5ba12)
